Makefile

```diff
@@ -0,0 +1,56 @@
+DC = docker exec -it opendorme_app_1 /bin/bash -c
+
+connect-app: ## Connect to app container
+	docker exec -it opendorme_app_1 /bin/bash
+
+build-env: ## Build local environment
+	mkdir -p ./storage/nginx
+	touch ./storage/nginx/error.log
+	touch ./storage/nginx/access.log
+	docker-compose build
+
+start-env: ## Start local environment silently
+	docker-compose up -d
+
+stop-env: ## Stop local environment
+	docker-compose down
+
+restart-env: stop-env start-env ## Restart the application
```

Comment:
can we remove the suffix `env` and just keep the command without suffix?
eg.
1. `make build-env` to `make build`
2. `make start-env` to `make start`
for better experience 💯

Reply:
Agree here. 🎉

Reply:
It will be updated 🤘🏻